_posts/2020-09-02-solidity-0.7.1-release-announcement.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.7.1 Release Announcement'
+date: '2020-09-02'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.7.1](https://github.com/ethereum/solidity/releases/tag/v0.7.1) .
+
+
+## Notable New Features 
+
+### Functions At File-Level
+
+Functions can now be defined at file-level. Such functions are also called "free functions"
```

Comment:
```suggestion
Functions can now be defined at file-level. Such functions are called "free functions"
```

Reply:
But we call them "file level function" all the time, too.

Reply:
Nowhere here, though, do we :-)? ``Functions At File-Level`` in the section header is not a name, but just says where the functions are...
If we *want* to call them "file level functions" all the time as well, then maybe this should just say ``Such functions are called "file level functions" or "free functions".`` :-). But yeah... maybe I'm nitpicking... I'd be fine with not changing this, it just seemed weird to me when I read it.